WildShape-Sheets-API/Services/AuthService.cs

```diff
@@ -40,26 +39,68 @@ public AuthService(UserService userService, DataBaseService dataBaseService, App
                     Console.WriteLine("Login valid");
                 } else {
                     Console.WriteLine("Login invalid");
-                    return null;
+                    return (null, null);
```

Comment:
The more standard approach would be to return the access and refresh tokens in a JSON structure. This makes it more flexible if you want to expand the authentication data later.